src/style.css

```diff
@@ -1,7 +1,53 @@
-.hello {
-  color: rgb(58, 107, 180);
+.main {
+  background-color: #f6f6f6;
+  display: flex;
+  justify-content: center;
 }
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add spacing(margin-top) at the top of your tasks container  to make it look smart.